data-prepper-api/src/main/java/com/amazon/dataprepper/metrics/MetricNames.java

```diff
@@ -26,9 +26,14 @@ private MetricNames() {}
     public static final String RECORDS_WRITTEN = "recordsWritten";
 
     /**
-     * Metric representing the number of records read from a buffer
+     * Metric representing the number of records read from a buffer but unchecked.
      */
-    public static final String RECORDS_READ = "recordsRead";
```

Comment:
Why are we removing this metric? Records written and records read seem like useful metrics.

Reply:
Added back-in